api/index.js

```diff
@@ -28,6 +28,7 @@ module.exports = async (req, res) => {
     theme,
     cache_seconds,
     custom_title,
+    lang,
```

Comment:
I think `locale` would be a better name for the prop since lang would be confusing since we have top-langs cards too.